server/internal/app/file_uploader.go

```
@@ -131,82 +131,89 @@
 
-		importData, assetsZip, pluginsZip, err := file_.UncompressExportZip(adapter.CurrentHost(ctx), f)
-		if err != nil {
-			return nil, errors.New("Fail UncompressExportZip :" + err.Error())
-		}
+			fs := afero.NewOsFs()
+			f, err := fs.Open(assembledPath)
+			if err != nil {
+				log.Printf("failed to open assembled file: %v", err)
+				return
+			}
+			defer f.Close()
```

golangci-lint (1.64.8)

Error return value of `f.Close` is not checked

(errcheck)